app/src/main/java/org/simple/clinic/util/Haversine.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.util
+
+import org.simple.clinic.location.Coordinates
+
+object Haversine {
```

Comment:
Why do we need a separate object for this when we can either have a top level function in, say, `LocationMath.kt`? Or have it as a function in the `Coordinates`class?

```kotlin
data class Coordinates(

    @ColumnInfo(name = "latitude")
    val latitude: Double,

    @ColumnInfo(name = "longitude")
    val longitude: Double
) {
  fun haversine(other: Coordinates): Distance
}
```

Reply:
`Coordinates#haversine()` sounds much better. I'll do this.

Reply:
I just realized that I've implemented Haversine in a later PR. Do you mind if I do this in that PR?

Reply:
Yeah, feel free to do that.

Reply:
This is done and you'll be able to see this in a future PR